src/pages/about.js

```diff
@@ -0,0 +1,51 @@
+import { graphql } from "gatsby"
+
+import React from "react"
+
+export default function About({ data }) {
+    const aboutData = data.prismicAboutPage.data
+
+    return (
+        <div>
+            <h1>TestTestTest</h1>
```

Comment:
Could you just ensure that all the data comes through correctly by utilizing the fields in the component?